modules/verifying-cluster-install-oci-agent-based.adoc

```diff
@@ -40,15 +40,15 @@ endif::pca[]
 
 ifndef::pca,c3[]
 [id="verifying-cluster-install-oci-agent-based_{context}"]
-= Verifying that your Agent-based cluster installation runs on {oci}
+= Verifying that your Agent-based cluster installation runs on {oci-distributed-no-rt}
```

Comment:
Is the verification step here for Edge or Distributed? Same question about line 45 below.